src/transformers/models/auto/tokenization_auto.py

```diff
@@ -328,7 +328,7 @@
             ),
             ("roc_bert", ("RoCBertTokenizer", None)),
             ("roformer", ("RoFormerTokenizer", "RoFormerTokenizerFast" if is_tokenizers_available() else None)),
-            ("rwkv", (None, "GPTNeoXTokenizerFast" if is_tokenizers_available() else None)),
+            ("rwkv", ("RWKVWorldTokenizer", None)),
```

Comment:
We can't really do this 😓 If one is supposed to be the "slow" we can just keep both. If they are fundamentally different we need something else.

Reply:
Alright, I'll fix it.

Reply:
You might be right, the model code of the recently implemented version for RWKV World model has changed compared to this one. I'll consider using a new folder.